bindings/rust/wasmedge-sys/src/lib.rs

```diff
@@ -238,6 +239,18 @@ pub type BoxedFn = Box<
         + Sync,
 >;
 
+pub type AsyncBoxedFn = Box<
+    dyn Fn(
+            &CallingFrame,
+            Vec<WasmValue>,
+            *mut std::os::raw::c_void,
+        ) -> Box<
+            dyn std::future::Future<Output = Result<Vec<WasmValue>, error::HostFuncError>> + Send,
+        > + Send
+        + Sync
+        + 'static,
+>;
+
```

Comment:
Please move the type alias declaration to the location just before line 265, as the type alias is paired with `ASYNC_STATE`.